test/framework/gardener_utils.go

```diff
@@ -125,6 +126,29 @@ func (f *GardenerFramework) CreateShoot(ctx context.Context, shoot *gardencorev1
 		return err
 	}
 
+	restConfig := f.GardenClient.RESTConfig()
+	versionedCoreClient, err := gardenversionedcoreclientset.NewForConfig(restConfig)
+	if err != nil {
+		f.Logger.Errorf("cannot create versioned client %s", err)
+		return err
+	}
+
+	binding := &gardencorev1beta1.Binding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: shoot.GetName(),
+		},
+		Target: corev1.ObjectReference{
+			Kind:       "Seed",
+			APIVersion: "core.gardener.cloud/v1beta1",
+			Name:       "local",
+		},
+	}
+
+	if _, err := versionedCoreClient.CoreV1beta1().Shoots(shoot.GetNamespace()).CreateBinding(ctx, shoot.GetName(), binding, metav1.CreateOptions{}); err != nil {
+		f.Logger.Errorf("cannot create binding for shoot %s: %s", shoot.GetName(), err)
+		return err
+	}
```

Comment:
Why do we need this? I like the removal of the `SeedName` field in `test/e2e/shoot/common.go`, but this should be enough to make `gardener-scheduler` find a seed and schedule it, right? We don't want to do it "manually" here, otherwise we don't include `gardener-scheduler` in the test?